extension/src/popup/views/Account/index.tsx

```diff
@@ -72,17 +74,10 @@ export const Account = () => {
     });
```

Comment:
❓ can you use the new sortBalances helper in `helpers/account.ts` to replace the lines above? (line 68-73)

Reply:
Argh, yes, good catch!